tests/python/contrib/test_ethosu/test_legalize.py

```diff
@@ -1502,5 +1503,61 @@ def verify(ext_func):
     verify(mod["tvmgen_default_ethos_u_main_0"])
 
 
+@pytest.mark.parametrize(
+    "ifm_shape,ifm_scale,ifm_zp,ofm_scale,ofm_zp",
+    [[(1, 8, 8, 3), 1.0, 0, 1.0, 0], [(1, 20, 30, 3), 1.345, 34, 0.32, -23]],
+)
+def test_ethosu_requantize(ifm_shape, ifm_scale, ifm_zp, ofm_scale, ofm_zp):
```

Comment:
Since requantize is a sub-pattern of other patterns -- we would need a test to ensure that such offloads are not affected in the presense of extra requantize ops.

Reply:
Done - I'm hoping I understood correctly, let me know if not!